integration-tests/deployment/ccip/changeset/cap_reg.go

```diff
@@ -7,8 +7,14 @@ import (
 	ccipdeployment "github.com/smartcontractkit/chainlink/integration-tests/deployment/ccip"
 )
 
+var _ deployment.Changeset = CapRegChangeSet
```

Comment:
I'm not sure what this is doing, other than catching a compile error if CapRegChangeSet isn't a ChangeSet.

Reply:
yeah its like this https://go.dev/doc/effective_go#blank_implements but for a function. Imo since its going cross package it helps document as well. Not a huge deal whether present or not